src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InteractiveWindow_InProc.cs

```diff
@@ -151,7 +152,15 @@ public string GetLastReplInput()
 
         public void Reset(bool waitForPrompt = true)
         {
-            ExecuteCommand(WellKnownCommandNames.InteractiveConsole_Reset);
```

Comment:
Why isn't this way stable? Is this actually hiding a bug where the user command won't be reliable?

Reply:
@tmat and I were not able to find a root cause looking through heap dumps all day. My best guess is we are somehow issuing the command at a point where `IsResetting` is true and it ends up getting dropped:

https://github.com/dotnet/interactive-window/blob/f45f6bef9e002b4b661ad7abf73f5dc996c0709d/src/Microsoft.VisualStudio.VsInteractiveWindow/VsInteractiveWindowCommandFilter.cs#L374

Reply:
Hmm, ExecuteCommand doesn't return a failure if we try executing it when the command is unavailable?

Reply:
The [documentation](https://docs.microsoft.com/en-us/dotnet/api/envdte._dte.executecommand?view=visualstudiosdk-2019) just says:

> `ExecuteCommand` cannot execute commands that are currently disabled in the environment. The `Build` method, for example, will not execute while a build is currently in progress.

The method returns `void` and doesn't declare any exceptions.